docs/source/en/api/pipelines/wuerstchen.md

```diff
@@ -0,0 +1,110 @@
+# Würstchen
+
+[Würstchen: Efficient Pretraining of Text-to-Image Models](https://huggingface.co/papers/2306.00637) is by Pablo Pernias, Dominic Rampas, and Marc Aubreville.
+
+The abstract from the paper is:
+
+*We introduce Würstchen, a novel technique for text-to-image synthesis that unites competitive performance with unprecedented cost-effectiveness and ease of training on constrained hardware. Building on recent advancements in machine learning, our approach, which utilizes latent diffusion strategies at strong latent image compression rates, significantly reduces the computational burden, typically associated with state-of-the-art models, while preserving, if not enhancing, the quality of generated images. Wuerstchen achieves notable speed improvements at inference time, thereby rendering real-time applications more viable. One of the key advantages of our method lies in its modest training requirements of only 9,200 GPU hours, slashing the usual costs significantly without compromising the end performance. In a comparison against the state-of-the-art, we found the approach to yield strong competitiveness. This paper opens the door to a new line of research that prioritizes both performance and computational accessibility, hence democratizing the use of sophisticated AI technologies. Through Wuerstchen, we demonstrate a compelling stride forward in the realm of text-to-image synthesis, offering an innovative path to explore in future research.*
+
+## Würstchen v2 comes to Diffusers
+
+After the initial paper release, we have improved numerous things in the architecture, training and sampling, making Würstchen competetive to current state-of-the-art models in many ways. We are excited to release this new version together with Diffusers. Here is a list of the improvements.
+
+- Higher resolution (1024x1024 up to 2048x2048)
+- Faster inference
+- Multi Aspect Resolution Sampling
+- Better quality
+
+## Text-to-Image Generation
+
+For the sake of usability Würstchen can be used with a single pipeline. This pipeline is called `WuerstchenPipeline` and can be used as follows:
+
+```python
+import torch
+from diffusers import WuerstchenPipeline
+
+device = "cuda"
+dtype = torch.float16
+num_images_per_prompt = 2
+
+pipeline = WuerstchenPipeline.from_pretrained(
+    "warp-diffusion/WuerstchenPipeline", torch_dtype=dtype
+).to(device)
+
+caption = "A captivating artwork of a mysterious stone golem"
+negative_prompt = "bad anatomy, blurry, fuzzy, extra arms, extra fingers, poorly drawn hands, disfigured, tiling, deformed, mutated"
+
+output = pipeline(
+    prompt=caption,
+    height=1024,
+    width=1024,
```

Comment:
It might be a good idea to comment about the limitation here a bit I think. How does this pipeline perform when the resolution is low (as low as 256x256) and high (as high as 1024x1024). 

Also, are we mentioning somewhere that we don't need more than 10 inference steps to get good-quality results to demonstrate the competitive advantage of Wuerstchen?

Reply:
I will do that in the documentation!